core/src/endpoints.ts

```diff
@@ -0,0 +1,32 @@
+import {
+	prodChains,
+	prodRelayKusama,
+	prodRelayPolkadot,
+} from "./endpoint_dump";
+
+export enum Ecosystem {
+	Polkadot = "Polkadot",
+	Kusama = "Kusama",
+	None = "None"
+}
+
+export interface IChainEndpoint {
+	name: string,
+	ecosystem: Ecosystem,
+	endpoints: Record<string, string>,
+}
+
+export let allChains: IChainEndpoint[] = [];
```

Comment:
@HoseinEmrani

Reply:
Is this related to the discussion we had about the chain related sub networks on the sidebar?